rotkehlchen/globaldb/minimized_schema.py

```diff
@@ -1,5 +1,5 @@
 # This file contains minimized db schema and it should not be touched manually but only generated by tools/scripts/generate_minimized_db_schema.py
-# Created at 2025-03-11 11:45:25 UTC with rotki version 1.37.2.dev444+gc78769ff5.d20250311 by yabirgb
+# Created at 2025-06-11 23:08:43 UTC with rotki version 1.39.1.dev28+g595e1afffb by banteg
```

Comment:
Undo this change. You need to run the script with only the userdb argument since this is the only one you changed. But anyway no need to rerun. Just undo this change here.